src/components/input/radio/button/style.css

```diff
@@ -0,0 +1,27 @@
+:host {
+	display: flex;
+	margin-right: 1em;
+	align-items: center;
+	position: relative;
+}
+
+:host:hover > * {
+	cursor: pointer;
+}
+
+:host input[type="radio"]:focus-visible + smoothly-icon {
+	outline: 3px solid rgb(var(--smoothly-primary-color));	
+}
+
+:host input[type="radio"] {
```

Comment:
Make the css targeting less specific to make it easier to target and change from the outside if needed.

I think `:host input {} `should be enough to target the correct element. If im wrong and `[type="radio"]` is necessary then keep it.

Same thing on line 12.